DESCRIPTION

```diff
@@ -6,15 +6,15 @@ Authors@R: c(
     person("Serge", "Smertin", email = "serge.smertin@databricks.com", role = c("aut", "cre")),
     person("Databricks", role = c("cph", "fnd"))
     )
-Description: Call all Databricks REST APIs via R
-License: Apache 2.0
+Description: Call all Databricks REST APIs from R.
+License: Apache License (>= 2)
 Encoding: UTF-8
 LazyData: true
 RoxygenNote: 7.2.3
 Roxygen: list(markdown = TRUE, roclets = c("rd", "namespace", "collate"))
 ByteCompile: false
 StagedInstall: no 
-Imports: httr, jsonlite, dplyr
+Imports: httr, jsonlite, dplyr, rlang
```

Comment:
what license does `rlang` have, or can it be considered as part of the R project?

Reply:
It’s MIT. But you were already using it and had a transitive dependency on it from dplyr.

Reply:
Also using this field doesn’t actually include the source of that package in your distributed packages so we don’t generally believe that the licenses need to be compatible (although your lawyers may have different feelings).  Some more details at https://r-pkgs.org/license.html#code-you-use.

Reply:
seems good then :) i'll separately start the short process for `vctrs` package intro